docs/upgrading/future.md

```diff
@@ -199,3 +199,70 @@ createBrowserRouter(routes, {
   },
 });
 ```
+
+## v7_skipActionStatusRevalidation
```

Comment:
We may need to copy this section to the `docs/upgrading/v6.md` file in the `v7` branch